testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AbstractBaseBrokerTest.java

```diff
@@ -263,7 +256,7 @@ protected void logInAsUserInIDPForFirstTime() {
 
     protected void logInAsUserInIDPForFirstTimeAndAssertSuccess() {
         logInAsUserInIDPForFirstTime();
-        assertLoggedInAccountManagement();
+        Assert.assertFalse(loginPage.isCurrent());
```

Comment:
Shouldn't this be updated to login to test app, and check that is opened rather the checking login page isn't current page